app/presenters/user_presenter.rb

```diff
@@ -15,11 +17,23 @@ def full_name
     "#{first_name} #{last_name}"
   end
 
-  def gravatar_id
-    Digest::MD5::hexdigest(email.downcase)
+  def joined
+    "Joined #{time_ago_in_words item.created_at} ago"
```

Comment:
The word "ago" is part of the attribute, but "Joined" is part of the framing of it and thus, probably belongs in the view.